mate-panel/panel.gresource.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gresources>
+  <gresource prefix="/org/mate/panel">
+    <file compressed="true">panel-properties-dialog.ui</file>
+    <file compressed="true">panel-run-dialog.ui</file>
+    <file alias="anim/wanda.png">../applets/fish/wanda.png</file>
```

Comment:
I think this one belongs to fish applet's resource. It's actually still installed to `/usr/share/mate-panel/pixmaps/` along with other images from that applet.

Reply:
This image was added here in g-p code 
https://git.gnome.org/browse/gnome-panel/diff/gnome-panel/nothing.cP?h=gnome-3-4&id=9fca67f7bf2db30838ba6dcf86d49ca965ed31e5
```
+ resource = g_strdup_printf ("%s%cand%c.png", "/org/gnome/panel/anim/", 'w', 'a');
```
as replacement for
```
- return g_strdup_printf ("%s/%cand%c.png", ICONDIR, 'w', 'a');
```
But nothing.cP with this code line was removed by @monsta here
http://git.mate-desktop.org/mate-panel/diff/mate-panel/nothing.cP?h=1.18&id=fbd0f1d8a815347beea1b2c5df5804c88aa406c6
No idea why you did that and where you  have moved the code for icon image?
So, it looks like we can remove this icon source from gresource file, as it does nothing.

Reply:
Because that was ancient easter egg which wasn't even enabled for GTK+3, and nobody ported it. But I see that `wanda.png` is still used in `org.mate.panel.applet.fish` schema.

Reply:
I missed this before PR was merged.

Reply:
history looks different here....
Let's say you missed it at the same time i pushed the button to merge it :)